openapi.yaml

```diff
@@ -12463,6 +12465,45 @@ components:
                 a relative URL, it is relative to the domain you retrieved the
                 Event from.
               example: /v4/support/tickets/11111
+        secondary_entity:
+          type: object
+          readOnly: true
+          description: |
+            Detailed information about the Event's secondary entity, which provides additional information
+            for `linode_boot`, `linode_create`, and `linode_clone` Event actions.
+          x-linode-cli-display: 5
+          properties:
+            id:
+              type: integer
+              description: |
+                The unique ID for an Event's secondary entity.
+
+
+                Secondary Entities are assigned the ID of the Linode they correspond to.
```

Comment:
This isn't true; this is the ID of the object that is the secondary entity (i.e. if I deploy a Linode from the official Debain9 image, the secondary entity's ID will be `linode/debain9` because that's the ID of that image).  They may have nothing to do with a Linode.